scripts/merge-and-build.sh

```diff
@@ -74,35 +74,6 @@ if [ "${OSE_VERSION}" != "3.2" ] ; then
   ssh-add ${HOME}/.ssh/id_rsa
 fi # End check if we are version 3.2
 
-function sanity_check() {
```

Comment:
I don't know if we have the best way to report errors from within the `GIT_SEQUENCE_EDITOR` -- a verification step that sits before it would let us signal errors better IMO. That flow would also allow us to not need to do error checks in the rebase script -- one thing I did not check in there was that no `[SQUASH]` commit existed earlier in time than the commit it was being squashed into.